tests/components/Dropdown.test.tsx

```diff
@@ -71,7 +71,58 @@ describe('Dropdown', () => {
     expect(inputNode).not.toHaveValue('item1');
   });
 
-  it('Handles option selection when pressing "Enter" on an option', () => {
+  it('takes tab navigation properly and updates focus on the option and input text', () => {
+    const dropdownProps: DropdownProps = {
+      screenReaderText: 'screen reader text here'
+    };
+    render(
+      <Dropdown {...dropdownProps}>
+        <DropdownInput />
+        <DropdownMenu>
+          <DropdownItem value='item1' focusedClassName='FocusedItem1'>
+            item1
+          </DropdownItem>
+        </DropdownMenu>
+      </Dropdown>
+    );
+    const inputNode = screen.getByRole('textbox');
+    userEvent.click(inputNode);
+    const itemNode = screen.getByText('item1');
+
+    userEvent.keyboard('{Tab}');
+    expect(itemNode.className).toContain('FocusedItem1');
+    expect(inputNode).toHaveValue('item1');
+
+    userEvent.keyboard('{Shift}{Tab}');
+    expect(itemNode.className).not.toContain('FocusedItem1');
+    expect(inputNode).not.toHaveValue('item1');
+  });
+
+  it('closes the dropdown menu when tabbing on last option', () => {
+    const mockedOnToggleFn = jest.fn();
+    const dropdownProps: DropdownProps = {
+      screenReaderText: 'screen reader text here',
+      onToggle: mockedOnToggleFn
+    };
+    render(
+      <Dropdown {...dropdownProps}>
+        <DropdownInput />
+        <DropdownMenu>
+          <DropdownItem value='item1' focusedClassName='FocusedItem1'>
+            item1
+          </DropdownItem>
+        </DropdownMenu>
+      </Dropdown>
+    );
+    const inputNode = screen.getByRole('textbox');
+    userEvent.click(inputNode);
+    userEvent.keyboard('{Tab}');
+    userEvent.keyboard('{Tab}');
```

Comment:
can this be combined into one keyboard event {Tab}{Tab}?

Reply:
done!